lib/suse/connect/client.rb

```diff
@@ -240,17 +252,20 @@ def registered?
       # Refreshing the service instead to remove the repos of deregistered product.
       def remove_or_refresh_service(service)
         if service.name == 'SMT_DUMMY_NOREMOVE_SERVICE'
+          log.info '-> Refreshing service ...'
           Zypper.refresh_all_services
         else
+          log.info '-> Removing service from system ...'
           System.remove_service service
         end
       end
 
       def print_success_message(product, action: 'Registered')
-        log.info "#{action} #{product.identifier} #{product.version} #{product.arch}"
+        log.info "\n#{action} #{product.identifier} #{product.version} #{product.arch}".bold
         log.info "Rooted at: #{@config.filesystem_root}" if @config.filesystem_root
         log.info "To server: #{@config.url}" if @config.url
         log.info "Using E-Mail: #{@config.email}" if @config.email
+        puts "==========\n"
```

Comment:
The double separator is a bit clunky between modules, can it only be outputted after the last module?

Reply:
I used it as a visual separator between the different products, but I can add this only to the final message (or get rid of the separators entirely and only use newlines?)

Reply:
One separator between products is fine, but I got two when registering SLE15: 
```
91e3485c3993:/tmp/connect # bin/SUSEConnect -d
==========
Deactivating sle-module-server-applications 15 x86_64 ...
-> Removing service from system ...
-> Removing release package ...

Deregistered sle-module-server-applications 15 x86_64
To server: https://scc.suse.com
==========
==========
Deactivating sle-module-basesystem 15 x86_64 ...
-> Removing service from system ...
-> Removing release package ...

Deregistered sle-module-basesystem 15 x86_64
To server: https://scc.suse.com
==========
Cleaning up ...
Successfully deregistered system.
```